core/src/main/java/com/datastax/oss/driver/api/core/data/CqlVector.java

```diff
@@ -84,7 +84,7 @@ public static <V extends Number> CqlVector<V> newInstance(List<V> list) {
    * </code> call on some CqlVector should return a CqlVector that is equal to the origin instance.
    *
    * @param str a String representation of a CqlVector
-   * @param subtypeCodec
+   * @param subtypeCodec the TypeCodec to parse the String representation
```

Comment:
This is true but it doesn't really tell us which string representation we're talking about :)

Vectors in CQL have a form that looks something like this:

```
org.apache.cassandra.db.marshal.VectorType(org.apache.cassandra.db.marshal.FloatType,3)
```

The first parameter there is the subtype of this vector, that is the type of values it contains.  The second is it's size parameter.

The subtypeCodec in CqlVector is intended to parse the string representation of the subtype so that we can build new instances of that subtype for the values contained by the vector.  So you probably want to say something like "the codec to parse the string representation of subtype values" here (or something similar) in order to make it very clear what values we're talking about.

Make sense?

Reply:
Thank you for your detailed explanation! Just changed to `the TypeCodec to parse the string representation of subtype values`.